src/d3d12.zig

```diff
@@ -258,8 +255,11 @@ pub const Surface = struct {
         _ = instance;
 
         if (utils.findChained(dgpu.Surface.DescriptorFromWindowsHWND, desc.next_in_chain.generic)) |win_desc| {
+            var hwnd: c.HWND = undefined;
+            @memcpy(std.mem.asBytes(&hwnd), std.mem.asBytes(&win_desc.hwnd));
```

Comment:
May still be worth having a `workaround issues with @alignCast panicking as these aren't real pointers` comment on this to clarify why it's being done this way?

Reply:
Good idea - fixed!